libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -77,6 +82,11 @@ public OrchestratorAdaptiveRecognizer(string modelFolder, string snapshotFile, I
             InitializeModel();
         }
 
+        [JsonIgnore]
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+        public bool ScoreEntities { get; set; } = false;
```

Comment:
Just wondering about usage - seems like InitializeModel() will override this depending if entity folder exists.

Reply:
Yeah, this is for unit testing (MockResolver, no real model/modelfolder)